data/ir-ads

```diff
@@ -0,0 +1,343 @@
+# List of ads and tracking company in Iran.
+# Thanks to PersianBlocker project
```

Comment:
giving credit is not enough, you CAN NOT copy content from a project that has a conflict license